include/onnxruntime/core/graph/graph.h

```diff
@@ -525,6 +525,10 @@ class Graph {
     return graph_inputs_including_initializers_;
   }
 
+  bool IsInputsIncludingInitializers(const NodeArg* node_arg) const noexcept{
```

Comment:
```suggestion
  // Return true if "node_arg" is a input or an initializer. Otherwise, returns false.
  bool IsInputsIncludingInitializers(const NodeArg* node_arg) const noexcept{
```

Reply:
comment added.

I am not against moving it to util. The main concern here is that, this should be consistant with graph object's graph_inputs_including_initializers_ variable. Having this method outside makes it hard to keep in sync with this.

---
In reply to: [429517645](https://github.com/microsoft/onnxruntime/pull/3996#discussion_r429517645) [](ancestors = 429517645)